atlas-geo-importer/templated-conda-env.yaml

```diff
@@ -0,0 +1,13 @@
+name: geo-importer
+channels:
+  - conda-forge
+  - ebi-gene-expression-group
+  - bioconda
+dependencies:
+  - magetab-curation-scripts
+  - perl-config-tiny
+  - perl-xml-parser
+  - perl-atlas-modules=0.3.1
```

Comment:
I am struggling to test this package.. 

I keep on getting 
```Solving environment: failed

ResolvePackageNotFound: 
  - perl-atlas-modules=0.3.1
 ```

p.s. I am on Mac with M1 chip

Reply:
are you using the correct [channel](https://anaconda.org/ebi-gene-expression-group/perl-atlas-modules)?

Reply:
I used the yaml file so should be using those channels...

There are three packages of ```perl-atlas-modules``` 
* version ```0.3.1``` in ```bioconda``` 
* version ```0.1.3``` and ```0.1.4``` in ```ebi-gene-expression-group```